src/Symfony/Component/HttpKernel/composer.json

```diff
@@ -44,7 +44,7 @@
         "symfony/translation-contracts": "^2.5|^3",
         "symfony/uid": "^5.4|^6.0|^7.0",
         "symfony/validator": "^6.4|^7.0",
-        "symfony/var-dumper": "^5.4|^6.4|^7.0",
+        "symfony/var-dumper": "^6.4|^7.0",
```

Comment:
this should be ^6.4.18 I guess, but can't we do without?

Reply:
I’m not sure :thinking: this is only needed for tests to pass.